app/serializers/user_base_serializer.rb

```diff
@@ -1,6 +1,6 @@
 class UserBaseSerializer < BaseSerializer
 
-  attributes :id, :email, :first_name, :is_admin, :last_name
+  attributes :id, :email, :first_name, :has_sidebar, :is_admin, :last_name
```

Comment:
Yeah, but I think since update sends back a user object, that's serialized with the base serializer, when we changet the user preferences using update, we wont' get the has_sidebar attribute unless we include it here.